### PR TITLE
fix(curriculum): separate empty string test in Sentence Analyzer

### PR DESCRIPTION
**Pull request by gikf**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

- - -
- Creates separate test for checking `getWordsCount` with _empty_ strings.
- While it falls under the _function should return the correct word count for any sentence_ case, edge case with _empty_ strings is not obvious from it.

### CODE REVIEW

**Review by majestic-owl448 — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/workshop-sentence-analyzer/66e2f06b191e305c00574e4d.md [hunk 1]
```diff
@@ -54,7 +54,13 @@ Your `getWordCount` function should return the correct punctuation count for any
 assert.strictEqual(getWordCount("freeCodeCamp has a great community of kind people"), 8);
 assert.strictEqual(getWordCount("The freeCodeCamp curriculum is constantly updated"), 6);
 assert.strictEqual(getWordCount("freeCodeCamp teaches both frontend and backend development"), 7);
+```
+
+Your `getWordCount` function should return the correct word count for empty string, or string only with spaces.
```
Comment (moT01) [inline curriculum/challenges/english/25-front-end-development/workshop-sentence-analyzer/66e2f06b191e305c00574e4d.md:59]:
```suggestion
Your `getWordCount` function should return the correct word count for an empty string, or a string only with spaces.
```

**Review by moT01 — APPROVED** [review]
